devicetypes/smartthings/motion-detector.src/motion-detector.groovy

```diff
@@ -85,11 +89,9 @@ def parse(String description) {
 }
 
 def isTracked() {
-	return device.getDataValue("model") == "MotionSensor51AU"
+	return isAuroraMotionSensor51AU()
 }
 
-def configure() {
```

Comment:
This device uses IAS Zone to report motion, right? I expect you need `enrollResponse` to get it enrolled so wondering why that's not here.

Reply:
In my self published DTH i used zigbee.iasZoneConfig(0,60) - enrollResponse() is called inside that method. Anyway, configure() method is removed here, because it doesn't make a difference (it wasn't in original DTH)